site.yml

```diff
@@ -187,7 +187,6 @@
 
 - name: ceph osds
   hosts: ceph_osds
-  serial: 1
```

Comment:
@chengtcli : Need to unit test this change with failed osd disk replacement scenario.